heat/core/tests/test_dndarray.py

```diff
@@ -500,7 +500,7 @@ def test_setitem_getitem(self):
         for c, i in enumerate(range(4)):
             self.assertEqual(a[1, c], i)
 
-        ####################################################
+        # ###################################################
```

Comment:
Small formatting hickup

Reply:
Scared the module :ghost:  Hickup is gone